### DISCUSSION TITLE
username taken but username-page gives me a 404?

### DISCUSSION THREAD

**Discussion by umlaeute** [community/community#63289] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i'm trying to setup an organization for `ftgl` development (a somewhat well known font-renderer for openGL that used to live on https://sourceforge.net/projects/ftgl/ but has been imported into some [personal](https://github.com/ulrichard/ftgl) [namespaces](https://github.com/frankheckenbach/ftgl) here on github a while ago).

now, when i try to register, github says that 
>  The name 'ftgl' is already taken. 

ok. so I try to find out more about who has taken that name and go to https://github.com/ftgl/ only to get a 404 error.

From https://docs.github.com/en/site-policy/other-site-policies/github-username-policy I understand that 
> Keep in mind that not all activity on GitHub is publicly visible; accounts with no visible activity may be in active use.

but shouldn't i get an error other than 404 in this case?
E.g. `401` (Unauthorized) would tell me that there *is* content but just not for me (as i haven't been granted access to a private group).

Or have i hit a bug, and the `ftgl` organization name would actually be still available?

**Comment by LOMASTER12309** [comment]
same
